### ISSUE TITLE
Rounded Edges for Default Popup

### ISSUE THREAD

**Issue by deshan-nawanjana** [GoogleChrome/chrome-extensions-samples#738]
Is there anyway we can have rounded edges for default popup?
It is better if we can do that using something like CSS "border-radius" :

body {
    width: 300px;
    height: 350px;
    border-radius: 15px;
}

**Comment by Shubham-Rasal** [comment]
Unfortunately, I don't think you can do it.

The frame around the popup page (highlighted in red in a graphics editor) is fully controlled by Chrome:

**Comment by M-SAI-SOORYA** [comment]
Yes, you can use CSS "border-radius" property to create rounded edges for default popups.
.popup {
  border-radius: 10px; 
  box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.5); 

}
In this example, the "popup" class is used to target the default popup element. The "border-radius" property is set to 10px to create rounded edges, and the "box-shadow" property is added to give the popup some depth.

You can adjust the value of the "border-radius" property to make the edges more or less rounded, depending on your preference.

**Comment by jpmedley** [comment]
I apologize for neglecting this for so long. I'm glad you finally got an answer. 

By they way, there is a Google group where you can ask questions and get faster answers. https://groups.google.com/a/chromium.org/g/chromium-extensions
